nMessenger/Source/Messenger/NMessengerViewController.swift

```diff
@@ -120,30 +120,33 @@ public class NMessengerViewController: UIViewController, UITextViewDelegate, NMe
         self.view.addSubview(self.messengerView)
     }
     /**
-     Creates InputBarView view and adds it to the view
+     Adds BaseInputBarView to the view
      */
     private func loadInputView()
     {
-        self.inputBarView = InputBarView(controller: self)
-        if let inputBarView = self.inputBarView as? UIView
-        {
-            self.view.addSubview(inputBarView)
-        }
+        self.inputBarView = self.customInputBar()
+        self.view.addSubview(inputBarView)
+    }
+    
+    /**
+     Methods sbould be ovverride if creating a custom input bar
```

Comment:
`Methods sbould be ovverride if creating a custom input bar` -> Spelling errors. Maybe `Override this method to create your own custom InputBarView`